lib/knapsack_pro/config/env.rb

```diff
@@ -293,6 +280,17 @@ def ci?
         def required_env(env_name)
           ENV[env_name] || raise("Missing environment variable #{env_name}")
         end
+
+        def knapsack_or_ci_env(knapsack_env_name, ci_env_name)
```

Comment:
The second arg is a method:

```suggestion
        def knapsack_or_ci_env(knapsack_env_name, ci_env_method)
```